manifest.yaml

```diff
@@ -1,154 +1,221 @@
+  # this is the image where data will go from 0.2.x
+  main:
+    type: data
+  # this is the image where files from the project asset directory will go
+  compat:
+    type: assets
+  # this is a pointer volume, where the image is specified in `<pointer-id>.volume-id` and the mount point is specificed in `main.mounts.<pointer-id>`
+  bitcoind:
+    type: pointer
+    package-id: bitcoind
+    volume-id: main
+    path: "/root"
+    readonly: true
+  btc-rpc-proxy:
+    type: pointer
+    package-id: btc-rpc-proxy
+    volume-id: main
+    path: "/root"
+    readonly: false
+  # TODO - update configurator to not gen cert
+  certificates:
+    type: certificate
+    # id of the volume you want the cert for
+    interface-id: main
+alerts:
+  install: |
+    READ CAREFULLY! LND and the Lightning Network are considered beta software. Please use with caution and do not risk more money than you are willing to lose. We encourage frequent backups. If for any reason, you need to restore LND from a backup, your on-chain wallet will be restored, but all your channels will be closed and their funds returned to your on-chain wallet, minus fees. It may also take some time for this process to occur.
+  uninstall: "READ CAREFULLY! Uninstalling LND will result in permanent loss of data, including its private keys for its on-chain wallet and all channel states. Please make a backup if you have any funds in your on-chain wallet or in any channels. Recovering from backup will restore your on-chain wallet, but due to the architecture of the Lightning Network, your channels cannot be recovered. All your channels will be closed and their funds returned to your on-chain wallet, minus fees. \n"
+  restore: |
+    Restoring LND will overwrite its current data, including its on-chain wallet and channels. Any channels opened since the last backup will be forgotten and may linger indefinitely, and channels contained in the backup will be closed and their funds returned to your on-chain wallet, minus fees.
+interfaces:
+  rest:
+    name: REST Interface
+    description: Specifies the interfaces to listen on for REST connections.
+    tor-config:
+      port-mapping:
+        8080: "8080"
+    ui: false
+    protocols:
+      - tcp
+      - http
+  watchtower:
+    name: Watchtower Interface
+    description: Specifies the interfaces to listen on for watchtower client connections.
+    tor-config:
+        port-mapping:
+          9911: "9911"
+    ui: false
+    protocols:
+      - tcp
+      - http
+  grpc:
+    name: gRPC Interface
+    description: Specifies the interfaces to listen on for gRPC connections.
+    tor-config:
+        port-mapping:
+          10009: "10009"
+    ui: false
+    protocols:
+      - tcp
+      - grpc
+  peer:
+    name: Peer Interface
+    description: Specifies the interfaces to listen on for p2p connections.
+    tor-config:
+        port-mapping:
+          9735: "9735"
+    ui: false
+    protocols:
+      - tcp
+      - http
+backup:
```

Comment:
this is a great default for most services, but IIRC lnd needs to take advantage of the custom backup feature, and only backup the channel list and on-chain wallet